ground/src/main/java/com/google/android/ground/ui/editsubmission/PhotoResult.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Note I updated this file to use the current year

Reply:
Thanks!